draft-ietf-httpbis-origin-frame.md

```diff
@@ -153,6 +153,20 @@ The Origin Set is also affected by the 421 (Misdirected Request) response status
 MUST create the ASCII serialisation of the corresponding request's origin (as per {{!RFC6454}},
 Section 6.2) and remove it from the connection's Origin Set, if present.
 
+Note:
+
+: Using an alternative service {{?RFC7838}} can result in using a service that has a different
+  port to the original service (that is, the server that a client contacts when it has no active
+  alternative services).  In that case, the initial origin will have a different port number.  A
+  client could be unable to send requests for the origin that advertised the alternative
+  service.  Explicitly including origins in the ORIGIN frame avoids this problem.
+
```

Comment:
I think you want to say something like this:

> Servers used as an alternative service {{?RFC7838}} will need to explicitly send the origin(s) that are available when sending ORIGIN frames, because the initial origin set (see {{ref}}) does not contain any alternative services. Because the origin includes the port number, this includes alternative services that are on the same host as the origin.

I.e., this is about more than just ports.